### PR TITLE
Add LEGO paper (IROS 19)

### PR DESCRIPTION
**Pull request by aditya-vk**
- [x] Update `.bib` file
- [x] Upload publication PDF to [Google Drive](https://drive.google.com/drive/folders/1M9fOGIIQ3e1R62dtVit5rZ5iWZqxfWV9)

### CODE REVIEW

File: siddpubs-conf.bib [hunk 1]
```diff
@@ -56,6 +56,13 @@ @string{cvpr
 
 % 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{kumar2019lego,
+ title={LEGO: Leveraging Experience in Roadmap Generation for Sampling-Based Planning},
+ author={Kumar, R. and Mandalika, A. and Choudhury, S. and Srinivasa, S.},
```
Comment (brianhou) [inline siddpubs-conf.bib]:
```suggestion
 author={Kumar, R. and Mandalika, A. and Choudhury, S. and Srinivasa, S.S.},
```

File: siddpubs-conf.bib [hunk 2]
```diff
@@ -56,6 +56,13 @@ @string{cvpr
 
 % 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{kumar2019lego,
+ title={LEGO: Leveraging Experience in Roadmap Generation for Sampling-Based Planning},
```
Comment (brianhou) [inline siddpubs-conf.bib]: This isn't going to be capitalized properly.

```suggestion
 title={{LEGO}: Leveraging Experience in Roadmap Generation for Sampling-Based Planning},
```

File: siddpubs-conf.bib (hunk 1 above)
Comment (aditya-vk) [reply siddpubs-conf.bib]: The official submission was made without the middle initial since that is how Sidd is registered on PaperPlaza.

Comment (aditya-vk) [reply siddpubs-conf.bib]: Since this is for our records, committing the suggestion.

**Review by brianhou — APPROVED** [review]

**Pull request merged**